reana_commons/tasks.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA common Celery tasks."""
+
+import logging
+
+from celery import shared_task
+from celery.task.control import revoke
+
+from reana_commons.api_client import JobControllerAPIClient
+
+log = logging.getLogger(__name__)
+
+
+@shared_task(name='tasks.stop_workflow',
+             ignore_result=True)
+def stop_workflow(workflow_uuid, job_list):
+    """Stop a workflow.
+
+    :param workflow_uuid: UUID of the workflow to be stopped.
+    :param job_list: List of job identifiers which where created by the given
+        workflow.
+    """
+    rjc_api_client = JobControllerAPIClient('reana-job-controller')
```

Comment:
Not a problem with this PR but `reana-job-controller` should be the only service the JobControllerAPIClient can be instantiated with. The class needs to implement __init__() and call super with `reana-job-controller`. Opening an issue